app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Login</h1>
+   <%= form_tag '/login' do %>
+   <%= label_tag :username%>
+   <%= text_field_tag :username %>
+   <%= label_tag :password%>
+   <%= password_field_tag :password%>
+   <%= submit_tag "Login"%>
+<%end%>
```

Comment:
@idprietop **Tip:** as a good coding practice, all the files should end within a blank line

Reply:
Thanks for the tip, I'll keep it in mind.